deegree-tests/pom.xml

```diff
@@ -89,6 +89,13 @@
             </goals>
           </execution>
         </executions>
+        <dependencies>
+          <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.15.1</version>
+          </dependency>
+        </dependencies>
```

Comment:
Adding a dependency here is a little unusual.

Reply:
Currently this dependency is required to run the deegree webapp on Jetty container for integration tests. Without the dependency the container fails to start the deegree webapp.
All dependencies need to be reviewed for final release version.